honeycombio/data_source_query.go

```diff
@@ -272,7 +272,7 @@ func extractFilter(d *schema.ResourceData, index int) (honeycombio.FilterSpec, e
 		filter.Value = vf
 		valueSet = true
 	}
-	vb, vbOk := d.GetOk(fmt.Sprintf("filter.%d.value_boolean", index))
+	vb, vbOk := d.GetOkExists(fmt.Sprintf("filter.%d.value_boolean", index))
```

Comment:
Can you clarify this change? `GetOkExists` is marked as deprecated.

Reply:
So `GetOk` is unable to differentiate between `value_boolean = false` and `value_boolean` being omitted completely. Not to say that there isn't a better solution than using a deprecated function, but the current usage of `GetOk` doesn't work with any filter with `value_boolean = false`

Reply:
Huh okay, I get it now. I find the implementation of `GetOk` a bit weird then tbh...
IMO It seems fine to use a deprecated function for now, eventually I'd like to refactor / improve all the `value_<type>` fields anyway.